hana/test/lean-draft.test.js

```diff
@@ -1,3 +1,4 @@
+require('../../test/cds.js')
```

Comment:
Why do we need to import our cds extensions here?

Reply:
When `cds` is not loaded in `describe` does not exist as a `global`

Reply:
can't we omit the wrapping describe in the hana test file instead?
don't see a big value as we mainly want to run the same tests as on sqlite.

Reply:
Other option would be to move `lean-draft` to `compliance/scenarios/sflight`. Would probably be more fitting as it is basically an `sflight` test suite and it would run using the symbolic link mechanism.

Reply:
probably makes more sense as effectively a single server start is sufficient for running the tests

Reply:
Will you do that before merging?